app/views/orders/_display_mutable_order_items.html.erb

```diff
@@ -8,11 +8,14 @@
   </tr>
   <% @order.order_items.each do |item| %>
     <tr>
-      <!-- !L this should use Lindsey's display_dollars helper once it exists! -->
-      <td><!--˅˄-->
-        <%= link_to "-", less_item_path(item), method: :patch %>
+      <td>
+        <% if item.quantity_ordered > 1 %>
+          <%= link_to "-", less_item_path(item), method: :patch %>
```

Comment:
It's fine if you prefer links, but I like to follow the rule that a link should only be a GET request, and a button should be any other kind of request.